DeDRM_plugin/ineptpdf.py

```diff
@@ -1893,32 +1583,13 @@ def initialize_standard(self, password, docid, param):
             self.decipher = self.decrypt_rc4  # XXX may be AES
         # aes
         if not set_decipher:
-            # This should usually already be set by now. 
+            # This should usually already be set by now.
             # If it's not, assume that V4 and newer are using AES
             if V >= 4:
                 self.decipher = self.decrypt_aes
         self.ready = True
         return
 
-    def verify_book_key(self, bookkey):
```

Comment:
Same as for EPUB - is it a good idea to remove this?

Reply:
> Same as for EPUB - is it a good idea to remove this?

Adobe Content Server 5 (ACS 5) DRM ，does any tools can decypt it ？ you can email me ab1307711@hotmail.com ，i will send you acsm file with adobe content server5 drm

Reply:
This PR probably can. But please don't use review comments for support requests.

Reply:
As above.